packages/maskbook/src/plugins/Ideamarket/UI/NotListed.tsx

```diff
@@ -0,0 +1,69 @@
+import { makeStyles, Typography, Button, Box } from '@material-ui/core'
+import type { UIProps } from '../types'
+
+const useStyles = makeStyles((theme) => ({
+    root: {
+        display: 'block',
+        width: '160px',
+        borderRadius: '10px',
+        boxShadow: '0px 0px 3px 1px #ADADAD',
+        marginRight: '10%',
+        overflow: 'hidden',
+        zIndex: 1,
+    },
+    topBox: {
+        backgroundColor: '#f7f7f7',
```

Comment:
Does this UI works in both light and dark mode?

Reply:
Yes, it does.